### PR TITLE
456 goodid modal

### PR DESCRIPTION
**Pull request by L03TJ3**
# Description

Modal + component for showing face-id sig

About # (link your issue here)
https://github.com/GoodDollar/GoodProtocolUI/issues/456

# Screens
As per requested UI
(desktop)
image

(mobile)
image

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [ ] I have followed all the instructions described in the initial task (check Definitions of Done)
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: dev-release [hunk 1]
```diff
@@ -61,12 +61,12 @@ const writePkg = async (pkg, json) => {
   console.log("5. Build design & login SDKs");
```
Comment (L03TJ3) [inline dev-release:61]: Temporarily disabling is needed for login-sdk.
Both Good-Design / Login-Sdk were not yet updated to latest version which has a hard requirement for typescript >5 (mentioned 2 weeks ago)

Good-Design I now tested and seems to work.

Login-sdk has to be retested but I did do this yet

Comment (johnsmith-gooddollar) [reply dev-release:61]: pls add some cmdline flag to skip login package

File: packages/good-design/src/advanced/customswitch/CustomSwitch.tsx [hunk 2]
```diff
@@ -1,92 +1,92 @@
...
+  isListOpen,
+  isLeft
+}: {
+  chain: string | undefined;
+  onPress: ((isLeft: boolean) => void) | ((isLeft: boolean, chain: string) => void);
+  isListItem: boolean;
+  isListOpen: boolean;
+  isLeft: boolean;
+}) => {
+  const type = isListItem ? "list" : "button";
```
Comment (johnsmith-gooddollar) [inline packages/good-design/src/advanced/customswitch/CustomSwitch.tsx:27]: the code is too "compressed" in this file 

please add newlines between all code blocks

File: packages/sdk-v2/src/sdk/claim/sdk.ts [hunk 3]
```diff
@@ -83,7 +83,11 @@ export class ClaimSDK extends BaseSDK {
       return url.toString();
     };
 
-    return { getLoginSig, getFvSig, getLink };
+    const deleteFvId = async () => {
+      return await this.deleteFVRecord();
```
Comment (johnsmith-gooddollar) [inline packages/sdk-v2/src/sdk/claim/sdk.ts:87]: no return await if not inside try catch

**Review by johnsmith-gooddollar — CHANGES_REQUESTED** [review]

**Comment by sirpy** [comment]
@L03TJ3 what is this?
I constantly see PRs without a description and without being connected to an issue

File: packages/good-design/src/advanced/customswitch/CustomSwitch.tsx (hunk 2 above)
Comment (L03TJ3) [reply packages/good-design/src/advanced/customswitch/CustomSwitch.tsx:26]: done

File: packages/sdk-v2/src/sdk/claim/sdk.ts (hunk 3 above)
Comment (L03TJ3) [reply packages/sdk-v2/src/sdk/claim/sdk.ts]: done

**Pull request merged**